configurator/templates/AppServer/configs.ini

```diff
@@ -0,0 +1,28 @@
+[SETTINGS]
+read_env_variables=false
+server=AppServer
```

Comment:
What's the purpose of this attribute "server". IMO this is not needed, a given configurator should only be able to configure a single server.